wls-briefwahl-service/src/test/java/de/muenchen/oss/wahllokalsystem/briefwahlservice/archunit/ArchUnitTest.java

```diff
@@ -31,8 +31,9 @@ void should_verifyArchUnitRuleForAllTestClassesOfService_when_running(final Argu
     }
 
     public static Stream<Arguments> allTestClassesRulesToVerify() {
-        return Stream.of(
+        return Stream.of(Arguments.of("RULE_BEFORE_EACH_NAMING_CONVENTION_MATCHED", MethodRules.RULE_BEFORE_EACH_NAMING_CONVENTION_MATCHED),
```

Comment:
sonst ist die rule doppelt
```suggestion
        return Stream.of(
```